requirements.txt

```
@@ -6,4 +6,5 @@ joblib
 pymemcache
 cloudpickle
 tqdm
-matplotlib
\ No newline at end of file
+matplotlib
+scipy==1.10.0
```

```suggestion
scipy>=1.7.0
```

The `highs` solver became the default starting from version [1.7.0](https://docs.scipy.org/doc/scipy-1.7.0/reference/reference/optimize.linprog-highs.html).

---

I think it's better to pin to a fixed version. We should do the same for all other packages btw.

---

I would agree if this was a standalone project, but this is library that is meant to be imported by others and we should not force them to update their dependencies especially if it isn't required.